test/test_selregparam.py

```diff
@@ -135,17 +135,17 @@ def test_ncp_value():
     "Check that the value returned by the NCP selection method is correct"
     
     loga = get_alpha_from_method('ncp')
-    logaref = 1.7574 # Computed with DeerLab-Matlab (0.9.2)
+    logaref = -9.18644 # Computed with DeerLab (1.0) grid method using quadprog

```

Comment:
This is a massive change compared to the previous value (1.76 to -9.19) - was the previous value wrong, or is there a problem with the new value? The other tests in this file also have new logaref values that are smaller than previously.